scripts/data_files/psa-arch-tests/pal_crypto_config.patch

```diff
@@ -0,0 +1,83 @@
+diff --git a/api-tests/platform/targets/tgt_dev_apis_stdc/nspe/pal_crypto_config.h b/api-tests/platform/targets/tgt_dev_apis_stdc/nspe/pal_crypto_config.h
+index dad40ec..8d19699 100644
+--- a/api-tests/platform/targets/tgt_dev_apis_stdc/nspe/pal_crypto_config.h
++++ b/api-tests/platform/targets/tgt_dev_apis_stdc/nspe/pal_crypto_config.h
```

Comment:
DES is disabled by default in v25.02, so no need to adjust the patch once we [remove DES](https://github.com/Mbed-TLS/TF-PSA-Crypto/pull/475).

Once https://github.com/Mbed-TLS/TF-PSA-Crypto/pull/475 is merged, this PR can remove the temporary definition of `PSA_KEY_TYPE_DES` in `crypto_compat.h`.

Reply:
> Once #475 is merged, this PR can remove the temporary definition of `PSA_KEY_TYPE_DES` in `crypto_compat.h`.

It's merged now :)